docs/internals/security.txt

```diff
@@ -27,8 +27,13 @@ implications, please send a description of the issue via email to
 team <https://www.djangoproject.com/foundation/teams/#security-team>`_.
 
 Once you've submitted an issue via email, you should receive an acknowledgment
-from a member of the security team within 48 hours, and depending on the
-action to be taken, you may receive further followup emails.
+from a member of the security team within 3 working days. After that, the
+security team will begin their analysis. Depending on the action to be taken,
+you may receive further followup emails. It can take several weeks before the
+security team has come to a conclusion. There is no need to chase the security
+team unless you discovery new, relevant information. All reports aim to be
+resolved within the industry-standard 90 days. Confirmed vulnerabilities with a
+:ref:`high severity level <severity-levels>` will be addressed promptly.
```

Comment:
Thought: Everyone's favourite: bug bounties. Does Django have one? If not, this might be the time to mention it as a guideline.

I recently added [a similar note for Wagtail's security policy](https://docs.wagtail.org/en/stable/contributing/security.html#bug-bounties). 

(If this is out of scope, I'm happy to open a separate PR).

Reply:
We do, it's described here: https://hackerone.com/django?type=team
We talk about it when asked, not sure if we should add it as a section in case it encourages more "beg bounties"

Reply:
> unless you discovery new

Typo: discover